app/models/plant.rb

```diff
@@ -0,0 +1,6 @@
+class Plant < ApplicationRecord
+  validates :image, :name, :species, :description, :light_requirements, :water_requirements, :when_to_plant, :harvest_time, :common_pests, presence: true
```

Comment:
Thanks for adding these fields! If for some reason we can't get any of these from the Trefle API, we can easily remove.